time/src/tokio.rs

```diff
@@ -0,0 +1,22 @@
+use std::{future::Future, time::Duration};
+
+use tokio::time::{
+    interval_at, sleep as tokio_sleep, timeout as tokio_timeout, Instant, Sleep, Timeout,
+};
+pub use tokio_stream::wrappers::IntervalStream as Interval;
+
+pub fn interval(period: Duration) -> Interval {
+    assert!(
+        period.as_millis() <= u32::MAX as u128,
+        "Period as millis must fit in u32"
+    );
```

Comment:
A comment why this is done would be good. I assume it's due to the restriction on [`gloo`'s `sleep` function](https://docs.rs/gloo/0.11.0/gloo/timers/future/fn.sleep.html):

> This function will panic if the specified `Duration` cannot be casted into a `u32` in milliseconds.